hadoop-hdds/docs/themes/ozonedoc/static/swagger-resources/recon-api.yaml

```diff
@@ -0,0 +1,1584 @@
+openapi: 3.0.0
+info:
+  title: Ozone Recon REST API
+  license:
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+    name: Apache 2.0 License
+servers:
+  - url: /api/v1/
+tags:
+  - name: Containers and Keys
+    description: APIs to fetch information about the available containers and keys. **Admin Only**
+  - name: Blocks Metadata
+    description: APIs to fetch metadata for the blocks available. **Admin Only**
+  - name: Namespace Metadata
+    description: APIs to fetch metadata for the namespace. **Admin Only**
+  - name: Cluster State
+    description: APIs to fetch data about the cluster state
+  - name: Datanodes
+    description: APIs to fetch data about the Datanodes
+  - name: Pipelines
+    description: APIs to fetch data about the Pipelines
+  - name: Tasks
+    description: APIs to fetch data about status of Recon Tasks
+  - name: Utilization
+    description: APIs to fetch data about space utilization
+  - name: Metrics
+    description: APIs to fetch data about various metrics from Prometheus
+    externalDocs:
+      description: Prometheus API docs
+      url: https://prometheus.io/docs/prometheus/latest/querying/api/
+paths:
+  /containers:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get all Container Metadata information
+      operationId: getContainerInfo
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ContainerMetadata'
+  /containers/{containerId}/keys:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get the Key metadata for all keys present in the given container ID
+      operationId: getContainerKeyMetadata
+      parameters:
+        - name: containerId
```

Comment:
Pls check this param name. Seems not correct.

Reply:
So in the API from the docs, we have the parameter as `:id` which is acting as a placeholder to denote the id of the container. We cannot however refer to this in the URL like that. But we can replace containerId with just id to be more in line with the documentation.
It was given as containerId for better readability.

Reply:
> So in the API from the docs, we have the parameter as `:id` which is acting as a placeholder to denote the id of the container. We cannot however refer to this in the URL like that. But we can replace containerId with just id to be more in line with the documentation. It was given as containerId for better readability.

If possible, make it inline with code, as Swagger yaml also works as for doc purposes as an open standard. And description of param may reflect the meaning of it.